bundles/org.openhab.binding.luxtronikheatpump/src/main/java/org/openhab/binding/luxtronikheatpump/internal/ChannelUpdaterJob.java

```diff
@@ -225,20 +227,33 @@ private void setExtendedState(Integer[] heatpumpValues, Integer[] heatpumpParams
         handleEventType(new StringType(longState), HeatpumpChannel.CHANNEL_HEATPUMP_STATUS);
     }
 
-    private void updateProperties(Integer[] heatpumpValues) {
+    public static Map<String, Object> getProperties(Integer[] heatpumpValues) {
+        Map<String, Object> properties = new HashMap<String, Object>();
+
         String heatpumpType = HeatpumpType.fromCode(heatpumpValues[78]).getName();
 
-        setProperty("heatpumpType", heatpumpType);
+        properties.put("heatpumpType", heatpumpType);
 
         // Not sure when Typ 2 should be used
         // String heatpumpType2 = HeatpumpType.fromCode(heatpumpValues[230]).getName();
-        // setProperty("heatpumpType2", heatpumpType2);
+        // properties.put("heatpumpType2", heatpumpType2);
+
+        properties.put("softwareVersion", getSoftwareVersion(heatpumpValues));
+        properties.put("ipAddress", transformIpAddress(heatpumpValues[91]));
+        properties.put("subnetMask", transformIpAddress(heatpumpValues[92]));
+        properties.put("broadcastAddress", transformIpAddress(heatpumpValues[93]));
+        properties.put("gateway", transformIpAddress(heatpumpValues[94]));
+
+        return properties;
+    }
 
-        setProperty("softwareVersion", getSoftwareVersion(heatpumpValues));
```

Comment:
private void setProperty method in this class is no longer used anywhere, could you drop it?

Reply:
yes. good find :+1: